database/index.js

```diff
@@ -20,23 +21,19 @@ db.once('open', () => {}); // connected
 
 const placesSchema = mongoose.Schema({
   id: {
-    type: String,
+    type: Number,
     unique: true,
   },
   name: String,
-  menu_url: String,
   address: String,
-  location: String,
   url: String,
   phone: String,
-  hours: [],
-  coords: {
-    lat: String,
-    lng: String,
-  },
+  hours: String,
+  lat: String,
+  long: String,
 });
 
-const Places = mongoose.model('Places', placesSchema);
+const Places = mongoose.model('restaurants', placesSchema);
```

Comment:
Not sure if it's a good practice to have your mongo db and collection be the same name. Could make debugging tricky.

http://mongoosejs.com/docs/models.html
"The first argument is the singular name of the collection your model is for. Mongoose automatically looks for the plural version of your model name."